<launch>
 	<node pkg="grid_map_visualization" type="grid_map_visualization" name="elevation_change_map_visualization" output="screen">
 		 <param name="grid_map_topic" value="/elevation_change_detection/elevation_change_map" />
		 <rosparam command="load" file="$(find elevation_change_detection)/config/visualization/elevation_change.yaml" />
    </node>
 	<node pkg="grid_map_visualization" type="grid_map_visualization" name="ground_truth_map_visualization" output="screen">
 		 <param name="grid_map_topic" value="/elevation_change_detection/ground_truth_map" />
         <rosparam command="load" file="$(find elevation_change_detection)/config/visualization/ground_truth.yaml" />
    </node>
</launch>